lib/moon/components/menu_item/switch.ex

```diff
@@ -0,0 +1,21 @@
+defmodule Moon.Components.MenuItem.Switch do
```

Comment:
Is this good name?

Reply:
was trying to name the component as the aria role from the list - https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/switch_role . looks like switch is the closest choice